stress/shell/src/main/java/alluxio/stress/cli/MultiOperationStressMasterBench.java

```diff
@@ -0,0 +1,502 @@
+import alluxio.conf.InstancedConfiguration;
+import alluxio.conf.PropertyKey;
+import alluxio.conf.Source;
+import alluxio.exception.AlluxioException;
+import alluxio.exception.status.InvalidArgumentException;
+import alluxio.grpc.CreateDirectoryPOptions;
+import alluxio.grpc.DeletePOptions;
+import alluxio.stress.StressConstants;
+import alluxio.stress.common.FileSystemClientType;
+import alluxio.stress.master.MasterBenchParameters;
+import alluxio.stress.master.MasterBenchTaskResultStatistics;
+import alluxio.stress.master.MultiOperationMasterBenchParameters;
+import alluxio.stress.master.MultiOperationMasterBenchTaskResult;
+import alluxio.stress.master.Operation;
+import alluxio.util.CommonUtils;
+import alluxio.util.FormatUtils;
+import alluxio.util.io.PathUtils;
+
+import com.google.common.collect.ImmutableList;
+import com.google.common.util.concurrent.RateLimiter;
+import org.HdrHistogram.Histogram;
+import org.apache.hadoop.conf.Configuration;
+import org.apache.hadoop.fs.FileSystem;
+import org.apache.hadoop.fs.Path;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.io.IOException;
+import java.net.URI;
+import java.util.Arrays;
+import java.util.List;
+import java.util.Random;
+import java.util.concurrent.Callable;
+import java.util.function.Function;
+
+/**
+ * Single node stress test.
+ */
+// TODO(jiacheng): avoid the implicit casts and @SuppressFBWarnings
+public class MultiOperationStressMasterBench
+    extends StressMasterBenchBase<MultiOperationMasterBenchTaskResult,
+    MultiOperationMasterBenchParameters> {
+  private static final Logger LOG = LoggerFactory.getLogger(MultiOperationStressMasterBench.class);
+
+  /**
+   * Creates instance.
+   */
+  @SuppressFBWarnings("BC_UNCONFIRMED_CAST")
+  public MultiOperationStressMasterBench() {
+    super(new MultiOperationMasterBenchParameters());
+  }
+
+  /**
+   * @param args command-line arguments
+   */
+  public static void main(String[] args) {
+    mainInternal(args, new MultiOperationStressMasterBench());
+  }
+
+  @Override
+  public String getBenchDescription() {
+    return String.join("\n", ImmutableList.of(
+        "A benchmarking tool to measure the master performance of Alluxio",
+        "This benchmark is a variant of StressMasterBench, where a user can specify multiple ",
+        "operations to run concurrently with some constraints to limit the throughput of ",
+        "each operations.",
+        "",
+        "Example:",
+        "# this would continuously run `GetFileStatus` and CreateFile "
+        + "for 30s and record the throughput after 5s warmup. "
+        + "The CreateFile throughput will be 4x more than GetFileStatus.",
+        "$ bin/alluxio runClass alluxio.stress.cli.MultiOperationStressMasterBench \\",
+        "--operations GetFileStatus,CreateFile --operations-ratio 1,4  --fixed-counts 100,1 \\",
+        "--warmup 1s  --duration 10s --in-process --client-type AlluxioNative --threads 16 ",
```

Comment:
`--in-process` is a hidden parameter and is supposed to be for internal uses only. Does it have to be specified so that the benchmark can run?

Reply:
yeah you are right this was the command i used for testing. removed the in-process flag.